misc/install.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+from subprocess import check_call
+
+
+def main(args: argparse.Namespace):
+    venv_args = ["uv", "venv"]
+    if args.python:
+        venv_args.extend(["--python", args.python, f".venv{args.python}"])
+    check_call(venv_args)
+    install_args = ["uv", "pip", "install", "-r", "test-requirements.txt", "-e", "."]
```

Comment:
move these to a dependency group in `pyproject.toml`?

Reply:
they are maintained upstream with `pip-compile`, kinda pointless to copy paste them into a proper uv config